paddlecloud/paddlejob/views.py

```diff
@@ -53,7 +55,8 @@ def post(self, request, format=None):
             psmemory = obj.get("psmemory", "1Gi"),
             topology = obj["topology"],
             gpu = obj.get("gpu", 0),
-            image = obj.get("image", "yancey1989/paddlecloud-job"),
+            image = obj.get("image", settings.JOB_DOCKER_IMAGE["image"]),
+            registry_secret = registry_secret,
```

Comment:
Check if we configured registry secrets, if not no need to provide this, using a public image.